havenapi/actions/registration_funnel.go

```diff
@@ -50,8 +53,43 @@ func RegistrationHandler(c buffalo.Context) error {
 	if err != nil {
 		return c.Error(500, err)
 	}
+	message := fmt.Sprintf("New registration for: %s", request.FormValue("email"))
+	_ = sendSlackNotification(message)
```

Comment:
Now that I look again in the morning I see this already works the way I was talking about in my review 🤦‍♂️ Nice work!